src/transformers/generation/flax_utils.py

```diff
@@ -1003,6 +1033,6 @@ def beam_search_body_fn(state, input_ids_length=1):
 
         # Take best beams for each batch (the score is sorted in descending order)
         sequences = flatten_beam_dim(sequences[:, :num_return_sequences, :])
-        scores = flatten_beam_dim(scores[:, :num_return_sequences])
+        sequences_scores = flatten_beam_dim(scores[:, :num_return_sequences]) if output_scores else None
 
-        return FlaxBeamSearchOutput(sequences=sequences, scores=scores)
+        return FlaxBeamSearchOutput(sequences=sequences, sequences_scores=sequences_scores)
```

Comment:
Likewise here let's just verify that we get the right scores out with one or two fast tests, and a slow integration test with Flax Whisper that shows the scores match those from PyTorch Whisper